File: Assets/Scripts/LogicObject/TankInfo.cs
class TankInfo
{
    public byte DriverID { get; set; } = 0xff;
    public bool IsDriven { get; set; } = false;
    public int Hp { get; set; } = 0;

    public static readonly TankInfo Instance = new TankInfo();

    private TankInfo()
    {
    }
}

File: server/logic/tankinfo.py
from network.configure import CONFIGURE


class TankInfo(object):
    position = CONFIGURE["TANK_INIT_POSITION"]
    turret_rotation = 0.0
    body_rotation = 0.0
    driver_id = 0xff
    is_driven = False
    hp = CONFIGURE["TANK_INIT_HP"]

    @classmethod
    def reset(cls):
        cls.position = CONFIGURE["TANK_INIT_POSITION"]
        cls.turret_rotation = 0.0
        cls.body_rotation = 0.0
        cls.driver_id = 0xff
        cls.is_driven = False
        cls.hp = CONFIGURE["TANK_INIT_HP"]
